README.md

```diff
@@ -13,6 +13,26 @@ protocol written in [Go](http://golang.org/) initially based on [gogopherd](http
 
     go get -u github.com/peterhellberg/gopher/cmd/gopherd
 
+## Compile
+
+	go build -o gopher.local.native cmd/gopherd/main.go
+
+## Usage
+For a standalone server use the following command to run the server on the localhost under port 7070.
+
+	./gopher.local.native \
```

Comment:
The name of the binary should be `gopherd`

Reply:
do you object to the longer names for the cross compiled versions?